examples/bank_reserves/bank_reserves/agents.py

```diff
@@ -10,15 +10,12 @@
     Northwestern University, Evanston, IL.
 """
 
-import mesa
-
 from .random_walk import RandomWalker
 
 
-class Bank(mesa.Agent):
-    def __init__(self, unique_id, model, reserve_percent=50):
-        # initialize the parent class with required parameters
-        super().__init__(unique_id, model)
```

Comment:
Why is the `super().__init__` removed? There could be other attributes initialized other than the UID and model.

Reply:
`mesa.Agent` `__init__` has
```python
self.model.agents_[type(self)][self] = None
```
without the `super().__init__`, the agent is not added to the `AgentSet`.

Reply:
I explained it briefly in the commit message:

> bank_reserves/charts: Don't make Bank an Agent
> It doesn't have a step function and there is only one, so it shouldn't be an Agent, just an Python object.

The idea is that the bank doesn't need to be an Agent, it can just be an Python object. No agent functionality is used, and it doesn't have a step function, so it can just be an object on which other agents interact on.

Reply:
Probably should be added in the docstring of the `Bank` class.

Reply:
That's a good point, I added an extended docstring note!